<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>M_SetFreqdlg</class>
 <widget class="QDialog" name="M_SetFreqdlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>设置频率</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../bin/image/607logo-6464.png</normaloff>../bin/image/607logo-6464.png</iconset>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>2</y>
     <width>322</width>
     <height>180</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;楷体&quot;;</string>
   </property>
   <property name="title">
    <string>采样频率设置</string>
   </property>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>16</y>
      <width>71</width>
      <height>157</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>91</width>
       <height>161</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>20</number>
      </property>
      <item>
       <widget class="QRadioButton" name="radioButton_single">
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;楷体&quot;;</string>
        </property>
        <property name="text">
         <string>单点</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_range">
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;楷体&quot;;</string>
        </property>
        <property name="text">
         <string>区域</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_load">
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;楷体&quot;;</string>
        </property>
        <property name="text">
         <string>导入</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>86</x>
      <y>16</y>
      <width>229</width>
      <height>157</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QComboBox" name="comboBox_end">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>50</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;楷体&quot;;</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_start">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>10</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;楷体&quot;;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_load">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>126</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;楷体&quot;;</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_loadname">
     <property name="geometry">
      <rect>
       <x>65</x>
       <y>126</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>128</y>
       <width>48</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;楷体&quot;;</string>
     </property>
     <property name="text">
      <string>文件名：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>92</y>
       <width>48</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;楷体&quot;;</string>
     </property>
     <property name="text">
      <string>采样数：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_end">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>52</y>
       <width>36</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;楷体&quot;;</string>
     </property>
     <property name="text">
      <string>结束：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>14</y>
       <width>36</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;楷体&quot;;</string>
     </property>
     <property name="text">
      <string>开始：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_start">
     <property name="geometry">
      <rect>
       <x>65</x>
       <y>12</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 9pt &quot;Arial&quot;;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_iter">
     <property name="geometry">
      <rect>
       <x>65</x>
       <y>89</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 9pt &quot;Arial&quot;;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_end">
     <property name="geometry">
      <rect>
       <x>65</x>
       <y>50</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 9pt &quot;Arial&quot;;</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>185</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>320</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton_update">
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;楷体&quot;;</string>
       </property>
       <property name="text">
        <string>更新</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_OK">
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;楷体&quot;;</string>
       </property>
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_cancel">
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;楷体&quot;;</string>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_expand">
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;楷体&quot;;</string>
       </property>
       <property name="text">
        <string>&gt;&gt;&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>175</width>
     <height>214</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;楷体&quot;;</string>
   </property>
   <property name="title">
    <string>当前频率</string>
   </property>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>16</y>
      <width>163</width>
      <height>190</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
